openslides/assignments/static/templates/assignments/slide_assignment.html

```diff
@@ -51,12 +51,15 @@ <h3 translate>Candidates</h3>
       <!-- votes -->
       <td ng-if="poll.has_votes">
         <div ng-init="votes = option.getVotes()">
-          <div ng-show="poll.yesnoabstain">
-            <span>
+          <div ng-show="poll.yesnoabstain||poll.yesno">
+            <span ng-show="poll.yesnoabstain">
               {{ votes[0].label }}: <strong>{{ votes[0].value }}</strong> ·
-              {{ votes[1].label }}: {{ votes[1].value }} ·
+              {{ votes[1].label }}: {{ votes[1].value }} .
```

Comment:
I do not understand the change in this line. What is the difference? `·` vs. `.`?